build/infrastructure/dash-charges-template.json

```
@@ -0,0 +1,427 @@
+{
+    "lenses": {
+      "0": {
+        "order": 0,
+        "parts": {
+          "0": {
+            "position": {
+              "x": 0,
+              "y": 0,
+              "colSpan": 6,
+              "rowSpan": 3
+            },
+            "metadata": {
+              "inputs": [],
+              "type": "Extension/HubsExtension/PartType/MarkdownPart",
+              "settings": {
+                "content": {
+                  "settings": {
+                    "content": "* **Incoming messages**: This is proportional to the amount of messages being sent to the domain.\n\n* **Number of active messages**: Large or growing numbers indicate that the infrastructure might not be scaled properly to handle the amount of load.\n\n* **Number of dead-lettered messages**: Dead-lettered messages are messages that the domain failed to handle. Each of these indicate a problem that needs to be resolved.",
+                    "title": "Service Bus Metrics",
+                    "subtitle": "",
+                    "markdownSource": 1,
+                    "markdownUri": null
+                  }
+                }
+              }
+            }
+          },
+          "1": {
+            "position": {
+              "x": 6,
+              "y": 0,
+              "colSpan": 5,
+              "rowSpan": 3
+            },
+            "metadata": {
+              "inputs": [
+                {
+                  "name": "sharedTimeRange",
+                  "isOptional": true
+                },
+                {
+                  "name": "options",
+                  "value": {
+                    "chart": {
+                      "metrics": [
+                        {
+                          "resourceMetadata": {
+                            "id": "/subscriptions/${subscription_id}/resourceGroups/${resouce_group_name}/providers/Microsoft.ServiceBus/namespaces/${service_bus_namespace}"
+                          },
+                          "name": "IncomingMessages",
+                          "aggregationType": 1,
+                          "namespace": "microsoft.servicebus/namespaces",
+                          "metricVisualization": {
+                            "displayName": "Incoming Messages"
+                          }
+                        }
+                      ],
+                      "title": "Incoming Messages (sum))",
```

If it was the first time I saw the dashboard, this (or something similar) would be a help to me.
```suggestion
                      "title": "Incoming Messages (sum per type))",
```

---

If you want that I guess you would also want (1) "sum per event type" or "sum per topic" and (2) the same addition in all 3 titles. Or...? Personally, I prefer it the way it is but I can change it.

---

I can live with it as is. Just a thought :)

---

Thx for input. I'll however leave it as-is.